src/cpg_seqr_loader/scripts/alphagenome_var_exp_script.py

```diff
@@ -0,0 +1,652 @@
+It loads variant data, applies a sliding window analysis to compare reference and alternate allele effects, identifies
+significant regions, generates summary tables, and optionally plots results for visualization.
+The script supports configurable parameters for
+input files, organs, thresholds, window size, merging, and output formats.
+"""
+
+from __future__ import annotations
+
+import argparse
+
+# ------------------------------------------------------------------
+# CLI
+# ------------------------------------------------------------------
+import io
+import warnings
+from pathlib import Path
+
+import gcsfs as gcsfs
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from alphagenome import colab_utils
+from alphagenome.data import gene_annotation, genome
+from alphagenome.data import transcript as transcript_utils
+from alphagenome.models import dna_client
+from alphagenome.visualization import plot_components
+from cloudpathlib.anypath import to_anypath
+from cpg_utils import config
+
+# In load_variants_table, if the input file is a GCS path, Path(path) will not work, and reading the file will fail.
+
+
+def saving_figure(path_logdir, fig):
+    buf = io.BytesIO()
+    fig.savefig(buf, format='png')
+    buf.seek(0)
+    with to_anypath(path_logdir).open('wb') as handle:
+        handle.write(buf.read())
+
+
+# defaults are brain, kidney, and nervous system, in that order
+organs = config.config_retrieve(
+    ['alphagenome_parm', 'organs'],
+    default=[
+        'UBERON:0000992',
+        'UBERON:0002371',
+        'UBERON:0000948',
+        'UBERON:0000955',
+        'UBERON:0001134',
+        'UBERON:0001264',
+    ],
+)
+threshold = config.config_retrieve(['alphagenome_parm', 'threshold'], default=0.5)
+min_length = config.config_retrieve(['alphagenome_parm', 'min_length'], default=1000)
+merge_distance = config.config_retrieve(['alphagenome_parm', 'merge_distance'], default=300)
+window_size = config.config_retrieve(['alphagenome_parm', 'window_size'], default=100)
+scan_span = config.config_retrieve(['alphagenome_parm', 'scan_span'], default=50000)
+plot_non_sig = config.config_retrieve(['alphagenome_parm', 'plot_non_sig'], default=False)
+scan_all_tracks = config.config_retrieve(['alphagenome_parm', 'scan_all_tracks'], default=True)
+epsilon = config.config_retrieve(['alphagenome_parm', 'epsilon'], default=1e-8)
+api_key = config.config_retrieve(['alphagenome_parm', 'api_key'], default=None)
+gtf = config.config_retrieve(
+    ['alphagenome_parm', 'gtf'],
+    default='gs://cpg-common-main/references/alphagenome/gencode.v46.annotation.gtf.gz.feather',
+)
+col_chrom = config.config_retrieve(['alphagenome_parm', 'col_chrom'], default='CHROM')
+col_pos = config.config_retrieve(['alphagenome_parm', 'col_pos'], default='POS')
+col_ref = config.config_retrieve(['alphagenome_parm', 'col_ref'], default='REF')
+col_alt = config.config_retrieve(['alphagenome_parm', 'col_alt'], default='ALT')
```

Comment:
personally I don't know why these are variables, these should be constants... what else would we call these values 😅